app/code/Magento/Cms/Test/Mftf/Section/BlockPageActionsSection.xml

```diff
@@ -20,5 +20,7 @@
         <element name="URLKey" type="input" selector="//div[@class='admin__form-field-control']/input[@name='identifier']"/>
         <element name="ApplyFiltersBtn" type="button" selector="//span[text()='Apply Filters']"/>
         <element name="blockGridRowByTitle" type="input" selector="//tbody//tr//td//div[contains(., '{{var1}}')]" parameterized="true" timeout="30"/>
+        <element name="delete" type="button" selector="//div[text()='{{var1}}']/parent::td//following-sibling::td[@class='data-grid-actions-cell']//a[text()='Delete']" parameterized="true"/>
+        <element name="deleteConfirm" type="button" selector=".action-primary.action-accept" timeout="60"/>
```

Comment:
Please rely on data attributes instead of CSS classes/ID attribute in selectors (see http://devdocs.magento.com/guides/v2.0/coding-standards/code-standard-demarcation.html#business-logic-and-javascript).